docs/configuration/transit-provider.md

```diff
@@ -0,0 +1,209 @@
+# Onboarding a new transit provider
+
+This guide describes the steps needed to add support for a new transit provider to the application and take them from testing to production.
+
+Before starting any configuration, the Cal-ITP team and transit provider staff should have a kickoff meeting to confirm that information provided is complete, implementation plan is feasible, and any approvals needed have been obtained.
+
+Then, the following steps are done by the Cal-ITP team to configure a new transit provider in the Benefits application.
+
+These steps currently assume the transit provider is using Littlepay as their transit processor. Support for integration with Switchio has been added but still needs to be documented, and integration with [other transit processors](https://www.camobilitymarketplace.org/contracts/) may be added in the future.
+
+
+## Pre-configuration prep work
+
+_These items can all be done in parallel._
+
+### Add transit provider to adoption table
+
+_Typically performed by the Benefits Product Manager._
+
+Open a Pull Request adding the new transit provider to the table in the README, similar to this [example](https://github.com/cal-itp/benefits/pull/2944).
+
+### Produce formatted transit provider logos
+
+_Typically performed by a designer._
+
+The application currently requires transit provider logos in the following sizes:
+
+- 296×144
+- 104×72
+
+These are currently displayed on white cards, so the logos themselves can be any color and the background can be white or transparent.
+
+### Update hardcoded parts of the application
+
+_Typically performed by an engineer._
+
+Certain parts of the codebase must be updated to support a new transit provider:
+
+- [`benefits/core/context/agency.py`](https://github.com/cal-itp/benefits/blob/main/benefits/core/context/agency.py) (requires a [migration](../../development/models-migrations/))
+- [`benefits/eligibility/views.py`](https://github.com/cal-itp/benefits/blob/main/benefits/eligibility/views.py)
+- [`benefits/enrollment/context/flow.py`](https://github.com/cal-itp/benefits/blob/main/benefits/enrollment/context/flow.py)
+- [Message files](../../development/i18n/)
+
+[Here is an example PR that makes these updates for a new transit provider.](https://github.com/cal-itp/benefits/pull/3141)
```

Comment:
given that #3141 is missing a translation, you're also free to link to https://github.com/cal-itp/benefits/pull/3246 now 😇